author.php

```diff
@@ -19,54 +19,128 @@
 		<div class="row">
 
 			<!-- Do the left sidebar check -->
-			<?php get_template_part( 'global-templates/left-sidebar-check' ); ?>
+<!--			--><?php //get_template_part( 'global-templates/left-sidebar-check' ); ?>
 
 			<main class="site-main" id="main">
 
-				<header class="page-header author-header">
+                <hr class="header-line">
+
+				<header class="page-header">
 
 					<?php
 					$curauth = ( isset( $_GET['author_name'] ) ) ? get_user_by( 'slug',
 						$author_name ) : get_userdata( intval( $author ) );
 					?>
 
-					<h1><?php esc_html_e( 'About:', 'understrap' ); ?><?php echo esc_html( $curauth->nickname ); ?></h1>
+                    <h3 class="entry-header">
```

Comment:
Can you bump this up to an `h1` or tweak the CSS so the text is centered vertically with the red line?